hangman.js

```diff
@@ -1 +1,125 @@
-console.log("welcome to hangman");
\ No newline at end of file
+var game = {
+  word: [],
+  hidden: [],
+  guesses: 5,
+
+  getWord: function(){
+    var temp = window.prompt("Have someone enter a secret word for you to begin the game.");
+    this.word = temp.toLowerCase().split("");
+    this.hidden = Array(this.word.length).fill(" - ");
+  },
+
+  getGuess: function(){
+    var message = "Word: " + this.hidden.join("");
+    message += "\nYou have " + this.guesses + " guesses.";
+    message += "\nEnter a letter.";
+
+    return window.prompt(message);
+  },
+
+  checkWin: function(){
+    if(this.word.toString() == this.hidden.toString())
+    {
+      return true;
+    }
+  },
+
+  indexesOf: function(letter)
+  {
+    var indexes = [];
+    for(var i = 0; i < this.word.length; i++)
+    {
+        if(this.word[i] == letter)
+        {
+            indexes.push(i);
+        }
+    }
+    return indexes;
+  },
+
+  playRound: function(){
+    var self = this;
+    var answer = this.getGuess().toLowerCase();
+    var goodGuess = false; // There's got to be a better way to do this
```

Comment:
this is a common pattern; setting a boolean value to act as a `flag` to control the flow of the loop. If you are intersted in a functional approach that increments a counter through side effects, I would recommend reviewing the [solutions](https://github.com/ga-wdi-exercises/hangman-lab/blob/jesse-review-solution/hangman.js)